repose-aggregator/components/filters/destination-router-filter/src/main/scala/org/openrepose/filters/destinationrouter/DestinationRouterFilter.scala

```diff
@@ -95,7 +91,13 @@ class DestinationRouterFilter @Inject()(configurationService: ConfigurationServi
             logger.error("The destination could not be added -- the destinations attribute was of an unknown type")
         }
 
-        mbcsRoutedReponse.mark(target.getId)
+        metricsServiceOption.foreach(metricsService =>
```

Comment:
I kind of feel like we'd be better served by having an optional of the metric itself. these blocks feel needlessly complicated as you interact with the registry every time. I see you are calling a mark method without an arguement, is that no longer available when dealing with the raw objects?